create_infotexts.py

```diff
@@ -374,15 +375,44 @@ def process_depicted_place(self, place_string, places_mapping):
                 place_as_wikitext = place_string
 
             # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
-                if places_mapping[place_string].get('commonscat'):
-                    self.content_cats.append(places_mapping[place_string]["commonscat"])
+            if places_mapping[place_string].get('commonscat') and \
+            places_mapping[place_string].get('commonscat') != "-" and \
+            places_mapping[place_string].get('commonscat') not in self.content_cats:
+
+                self.content_cats.append(places_mapping[place_string]["commonscat"])
 
         else:
             self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
             place_as_wikitext = place_string
 
 
+        for place in places_mapping:
+            if place.lower() in desc_string.lower():
+
+                if places_mapping[place]["wikidata"] and places_mapping[place]["wikidata"] != "-":
+                    if place_as_wikitext: # There is already a depicted place found in field <Ort, foto>
```

Comment:
Rather than having to have two separate methods depending on if a second is found it might be simpler to just use a list and do `'/'.join(...)` after the loop.

But an important question is whether it ever makes sense to have two places? (My gut feeling says this probably means one or both are wrong)